### PR TITLE
Return state suspended for all suspended runtimes

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- computing runtimes endpoint Satus.State value

**Related issue(s)**

### CODE REVIEW

**Review by ebensom — APPROVED** [review]

File: components/kyma-environment-broker/internal/runtime/converter.go [hunk 1]
```diff
@@ -167,4 +174,48 @@ func (c *converter) ApplyUpdateOperations(dto *pkg.RuntimeDTO, oprs []internal.U
...
+			dto.Status.State = pkg.StateUpdating
+		}
+	default:
+		dto.Status.State = pkg.StateSucceeded
+	}
+
+	if dto.Status.Suspension != nil && dto.Status.Suspension.Count > 0 {
+		// there is no unsuspension operation or the suspension is started after last unsuspension
+		if dto.Status.Unsuspension == nil ||
+			(dto.Status.Unsuspension.Count > 0 && dto.Status.Unsuspension.Data[0].CreatedAt.Before(dto.Status.Suspension.Data[0].CreatedAt)) {
```
Comment (PK85) [inline components/kyma-environment-broker/internal/runtime/converter.go:212]: double check if Data[0] has newest entries

Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/internal/runtime/converter.go:212]: listing operations in the DB is ordered by date

**Review by PK85 — APPROVED** [review]

**Pull request merged**